### DISCUSSION TITLE
I encountered some confusion while reading the Milvus Docs

### DISCUSSION THREAD

**Discussion by TongTong313** [milvus-io/milvus#25464] [Q&A and General discussion] [answered]
In the section of User Guide -> Search and Query -> Search, the description of "limit" and "offset" are **inconsistent** in "prepare search parameters" and "conduct a vector search".  In "prepare search parameters", the description of "limit" is  "Number of entities to skip during the search" but in "conduct a vector search", the desrciption of "limit" is "Number of the most similar results to return." The same inconsistency also appears in the 'offset' parameter. 

Are there any mistakes here, or is my understanding of these two parameters not accurate enough?

<img width="467" alt="9fb3aa897dd82581a6072775f711dee" src="https://github.com/milvus-io/milvus/assets/116777179/1f9a15af-9a0f-4577-b3f2-dc79e7fe5761">
<img width="467" alt="65f7dbf0c3dc3e67f568302111b6453" src="https://github.com/milvus-io/milvus/assets/116777179/562daf17-1597-4812-a282-e7361b08e4c4">

**Comment by xiaofan-luan** [comment, marked as answer]
it's. actaully wrong.
limit is the number of vector to return and offset is the number of vector to skip.
